modules/appeals_api/app/services/appeals_api/pdf_construction/higher_level_review/v4/structure.rb

```diff
@@ -0,0 +1,177 @@
+# frozen_string_literal: true
+
+require 'prawn/table'
+
+module AppealsApi
+  module PdfConstruction
+    module HigherLevelReview::V4
+      class Structure
+        MAX_ISSUES_FIRST_PAGE = 7
+        MAX_ISSUES_SECOND_PAGE = 6
+        MAX_ISSUES_ON_MAIN_FORM = (MAX_ISSUES_FIRST_PAGE + MAX_ISSUES_SECOND_PAGE).freeze
+        SHORT_EMAIL_THRESHOLD = 100
+        DEFAULT_TEXT_OPTIONS = { overflow: :shrink_to_fit, min_font_size: 8, valign: :bottom }.freeze
+
+        def initialize(higher_level_review)
+          @higher_level_review = higher_level_review
+        end
+
+        def form_title
+          '200996_v4'
+        end
+
+        def form_fields
+          @form_fields ||= FormFields.new
+        end
+
+        def form_data
+          @form_data ||= FormData.new(higher_level_review)
+        end
+
+        def form_fill
+          FormFields::FIELD_NAMES.keys.reduce({}) do |values, field_name|
+            values.merge({ form_fields.send(field_name) => form_data.send(field_name) })
+          end.merge(fill_contestable_issues_dates).merge(fill_benefit_type_fields)
+        end
+
+        def insert_overlaid_pages(form_fill_path)
+          pdftk = PdfForms.new(Settings.binaries.pdftk)
+          output_path = "/tmp/#{higher_level_review.id}-overlaid-form-fill-tmp.pdf"
+          temp_path = fill_autosize_fields
+          pdftk.multistamp(form_fill_path, temp_path, output_path)
+          output_path
+        end
+
+        def add_additional_pages
+          return unless additional_pages?
+
+          @additional_pages_pdf ||= Prawn::Document.new(skip_page_creation: true)
+          Pages::AdditionalIssues.new(@additional_pages_pdf, form_data).build!
+          @additional_pages_pdf
+        end
+
+        def final_page_adjustments
+          # no-op
+        end
```

Comment:
Is this method necessary?

Reply:
Yes, each specific form like this (new VA HLR v4 form) is called by by our generic AppealsApi::PdfConstruction::Generator calls this for all of our specific forms implementations like this one.